CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 ### New features
 
+* Add new cop `Rails/RedundantAllowNil` that checks for cases when `allow_blank` makes `allow_nil` unnecessary in model validations. ([@elebow][])
```

Comment:
Please move this to the proper section of the changelog. 0.65 was released a while ago.